tests/binary_agreement.rs

```diff
@@ -27,31 +27,139 @@ extern crate threshold_crypto as crypto;
 
 mod network;
 
+use std::collections::BTreeMap;
 use std::iter::once;
 use std::sync::Arc;
 
 use rand::Rng;
 
-use hbbft::binary_agreement::BinaryAgreement;
-use hbbft::NetworkInfo;
+use hbbft::binary_agreement::sbv_broadcast;
+use hbbft::binary_agreement::{BinaryAgreement, Message, MessageContent};
+use hbbft::{NetworkInfo, Target, TargetedMessage};
 
-use network::{Adversary, MessageScheduler, NodeId, SilentAdversary, TestNetwork, TestNode};
+use network::{
+    Adversary, MessageScheduler, MessageWithSender, NodeId, SilentAdversary, TestNetwork, TestNode,
+};
```

Comment:
We're currently in the process of transitioning from the old (`network`) to the new (`net`) test framework. I'm not sure how complicated it would be to port this to the new one?